docs/index.md

```diff
@@ -29,6 +29,7 @@ information [here](https://innovation.microsoft.com/en-us/exploring-ai-at-scale)
 
 # What's New?
 * [2021/03/08] [ZeRO-3 Offload: Scale your models to trillion parameters without code changes while leveraging both CPUs & GPUs](https://www.deepspeed.ai/news/2021/03/07/zero3-offload.html)
+* [2021/03/04] [1-bit Adam v2: NCCL-based implementation and more](https://www.deepspeed.ai/tutorials/onebit-adam/)
```

Comment:
same here, let's update the date and move it top

Reply:
Resolved, thanks!